### DISCUSSION TITLE
Browser setting for clicking on hyperlink urls

### DISCUSSION THREAD

**Discussion by mortenivar** [kovidgoyal/kitty#6786] [Q&A] [answered]
When clicking on a hyperlink url in the kitty terminal, it is opened in firefox, but I would like to see it in a different browser. I have the $BROWSER variable set but this is not honored, apparently. I think I have looked everywhere, but I can't find the relevant setting. How do I configure what browser to use?

**Comment by kovidgoyal** [comment, marked as answer]
By default, kitty will use your systems URL opener which on linux is
xdg-open and on macos is open. And if you want to control things in much
more detail, use https://sw.kovidgoyal.net/kitty/open_actions/

**Comment by mortenivar** [comment]
Thank you. Actually my preferred browser was seen as the default browser by the xdg-settings program. It seems to honor the $BROWSER variable. However, regardless of that, xdg-open still used firefox for everything. I solved it by editing the
~/.config/mimeapps.list and now it works. I have a feeling, though, that maybe the mimeapps.list should not be hand-edited ...

**Reply by SolitudeSF** [reply]
usually you can do that with `xdg-mime default firefox.desktop x-scheme-handler/https`, or whatever scheme you want to associate it with

**Reply by mortenivar** [reply]
I see, thank you.
